plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/procedure/MigrationUtils.java

```diff
@@ -109,7 +109,7 @@ public static List<DataFile> buildDataFiles(
             FileEntry file = files.next();
             String fileLocation = file.location().toString();
             String relativePath = fileLocation.substring(location.length());
-            if (relativePath.contains("/_") || relativePath.contains("/.")) {
+            if (relativePath.contains("/_") || fileLocation.contains("/.")) {
```

Comment:
This change seems wrong. Hive doesn't ignore files where the parent directory starts with `.` as far as I know. 
The reason we used `relativePath` here was ignoring files **_under_** the table location. 
You can investigate Hive behavior with `S3HiveQueryRunner`.

Reply:
I understand that hive doesn't ignore files where the parent directory starts with `.`. 
Currently the api `add_files` just has the path to load the data. It does not have the information of source table or its path which can be used to verify path belongs under source table or not.  Should we add that information in the api ? OR
Instead of checking `.` in `fileLocation`, we can check it at current directory.

Reply:
@ebyhr let me know your views

Reply:
Did you confirm Spark behavior? You can use the following command:
```sql
testing/bin/ptl env up --environment EnvSinglenodeSparkIceberg
```
and then:
```
docker exec -it ptl-spark spark-sql
```